pandas/tests/extension/base/ops.py

```diff
@@ -113,14 +113,13 @@ def test_add_series_with_extension_array(self, data):
         expected = pd.Series(data + data)
         self.assert_series_equal(result, expected)
 
-    @pytest.mark.parametrize("box", [pd.Series, pd.DataFrame])
```

Comment:
im on board with this, but in the past @jorisvandenbossche has objected bc it would require library authors to re-implement the fixture

Reply:
Ah good point. I didn't realize this was a base extension test so I'll exclude this one.